.puppet-lint.rc

```diff
@@ -0,0 +1 @@
+--no-relative_classname_inclusion-check
```

Comment:
This file needs rebased

Reply:
I took it from #221. In one of your comments (https://github.com/ghoneycutt/puppet-module-pam/pull/221#discussion_r436889223) you said:
> Please change to ">= 4.25.0 < 7.0.0"